app/main.py

```diff
@@ -1,2 +1,56 @@
+from typing import Any, Hashable
+
+
 class Dictionary:
-    pass
+    def __init__(self,
+                 initial_capacity: int = 8,
+                 load_factor: float = 2 / 3
+                 ) -> None:
+        self.initial_capacity = initial_capacity
+        self.load_factor = load_factor
+        self.size = 0
+        self.hash_table = [None] * self.initial_capacity
+        self.max_capacity = self.initial_capacity * self.load_factor
+
+    def __setitem__(self, key: Hashable, value: Any) -> None:
+
+        index, hash_of_the_key = self.find_available_index(key)
+        if not self.hash_table[index]:
+            if self.size >= self.max_capacity:
+                self.resize()
+            index, hash_of_the_key = self.find_available_index(key)
+            self.size += 1
+        self.hash_table[index] = [key, hash_of_the_key, value]
```

Comment:
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
        index, hash_of_the_key = self.find_available_index(key)
        if not self.hash_table[index]:
            if self.size >= self.max_capacity:
                self.resize()
            index, hash_of_the_key = self.find_available_index(key)
            self.size += 1
        self.hash_table[index] = [key, hash_of_the_key, value]
```

why do you need an empty line here?)

Reply:
Beautiful. I gonna fix it)